### PR TITLE
Add mute feature

### PR DESCRIPTION
**Pull request by geekchick**
-For this [Jira ticket](https://jira.vonage.com/browse/DEVX-5601)
- Added new active flag, excludedStreams, and excludedStreamIds to mute function, with a change in functionality in the mute function
- Added new test for excludedStreams

### CODE REVIEW

File: opentok/opentok.py [hunk 1]
```diff
@@ -1422,30 +1424,45 @@ def __init__(
     
 
 
-    def mute(self, session_id: str, stream_id: str= "", options: dict = {}) -> requests.Response:
+    def mute(self,
```
Comment (jeffswartz) [inline opentok/opentok.py:1427]: @geekchick Let's change this to two methods: `mute_all(session_id, active, excludedStreamIds)` and `mute_stream(session_id, stream_id`. 

What is the `data` parameter? Can we remove it?

**Review by jeffswartz — CHANGES_REQUESTED** [review]

File: opentok/opentok.py (hunk 1 above)
Comment (geekchick) [reply opentok/opentok.py:1646]: Hi @jeffswartz the data parameter is the body or payload for the request. It gets defined as a dictionary in the method signature and used in the code to populate the request body.

File: opentok/opentok.py [hunk 2]
```diff
@@ -1639,46 +1641,88 @@ def __init__(
...
+            elif response.status_code == 404:
+                raise NotFoundError("The session or a stream is not found.")
+        except Exception as e:
+            raise OpenTokException(
+                ("There was an error thrown by the OpenTok SDK, please check that your session_id {0} and excludedStreamIds (if exists) {1} are valid").format(
+                    session_id, excludedStreamIds))
+
+        
+
+    def mute_stream(self, session_id: str, stream_id: str, options: dict = {}) -> requests.Response:
```
Comment (jeffswartz) [inline opentok/opentok.py:1694]: See my comment about the `options` parameter of the `mute_all()` method.

File: opentok/opentok.py [hunk 3]
```diff
@@ -1639,46 +1641,88 @@ def __init__(
             app_version=app_version
         )
 
+    def mute_all(self, 
+                session_id: str, 
+                excludedStreamIds: Optional[List[str]], 
+                active: bool= True, 
+                options: dict = {}) -> requests.Response:
```
Comment (jeffswartz) [inline opentok/opentok.py:1648]: Can we remove this parameter? We do not include this options parameter in other methods. Can the method use a private variable internally that populates the options? I don't think we should include a parameter that the developer will not use.

**Review by jeffswartz — CHANGES_REQUESTED** [review]

File: opentok/opentok.py (hunk 3 above)
Comment (geekchick) [reply opentok/opentok.py:1648]: Hi @jeffswartz ! Sure, I can definitely remove the options parameter in the function.

File: opentok/opentok.py (hunk 2 above)
Comment (geekchick) [reply opentok/opentok.py:1694]: Ok, will do!

**Review by jeffswartz — APPROVED** [review]

**Pull request merged**